eva-lib/src/main/java/uk/ac/ebi/eva/lib/repository/AnnotationMetadataRepository.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.lib.repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+
+import uk.ac.ebi.eva.commons.models.metadata.AnnotationMetadata;
+
+import java.util.List;
+
+/**
+ * Spring MongoRepository for querying collection of AnnotationMetadata documents
+ */
+public interface AnnotationMetadataRepository extends MongoRepository<AnnotationMetadata, String> {
+
+    /**
+     * Query for all AnnotationMetadata from collection, ordered first by cache version descending and then vep version
+     * descending.
+     *
+     * @return List of AnnotationMetadata objects
+     */
+    List<AnnotationMetadata> findAllByOrderByCacheVersionDescVepVersionDesc();
```

Comment:
Yeah, confirmed by both @garysaunders and @cyenyxe . The updates to the cache are likely to affect the annotation more than the updates to the tool, and updates to the cache are more likely to be changes to the fundamental data.